scripts/build.ps1

```diff
@@ -110,16 +110,19 @@ try {
   Create-Directory -Path $LogDir
 
   if ($architecture -ne "") {
-    $DOTNET_SKIP_FIRST_TIME_EXPERIENCE = 1
+    $env:DOTNET_CLI_TELEMETRY_OPTOUT = 1
```

Comment:
This wasn't be respected if it wasn't properly exported. Will also submit a fix for LLVMSharp.